server/base/src/main/java/org/apache/accumulo/server/manager/LiveTServerSet.java

```diff
@@ -393,6 +404,12 @@ public synchronized Set<TServerInstance> getCurrentServers() {
     return new HashSet<>(currentInstances.keySet());
   }
 
+  public synchronized Map<String,Set<TServerInstance>> getCurrentServersGroups() {
+    Map<String,Set<TServerInstance>> copy = new HashMap<>();
```

Comment:
Its possible that depending on the uses patterns that it may be more efficient to compute an immutable map of this info when it changes.  The tserver does this for online tablets in https://github.com/apache/accumulo/blob/main/server/tserver/src/main/java/org/apache/accumulo/tserver/OnlineTablets.java

I don't know if this would be useful w/o analyzing the larger way in which this is used.